### PR TITLE
Awesome Books

### PR DESCRIPTION
**Pull request by cyrilmuchemi**
- This is a simple HTML website for testing various javaScript Skills.
- My partner @SiansandaTamara and I, have implemented javascript methods and objects to add and remove books from this simple app.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  static addBook(book) {
+    const books = Store.getBooks();
+
+    books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(author) {
```
Comment (topeogunleye) [inline app.js:63]: - [ ] [REQUIRED] I can see that your method to remove books from local storage is not working. Please kindly fix this

https://user-images.githubusercontent.com/52934128/164044395-0c0c6272-a36b-4d02-bcbb-63bc5e0f206f.mp4

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @cyrilmuchemi,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please follow the wireframe and put the booklist before the form
> Screenshot from 2022-04-19 16-52-03
> Wireframe:
> book-form

> ## [Optional] suggestions:
> * Nothing to mention
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.js (hunk 1 above)
Comment (Kingstalux) [reply app.js:63]: resolved

Comment (cyrilmuchemi) [reply app.js:63]: @Kingstalux , what's the review?

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✅ App works as expected 🔥 
✅ Linters are passing 👏 
✅ Good commit messages 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @cyrilmuchemi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (topeogunleye) [reply app.js:64]: Great 👍🏿

**Pull request merged**